server/views/layout/default.njk

```diff
@@ -19,6 +19,7 @@
     <link rel="manifest" href="/assets/icons/manifest.json"/>
     <link rel="mask-icon" href="/assets/icons/safari-pinned-tab.svg" color="#000000"/>
     <script src="/assets/libs/picturefill.min.js" async></script>
+    <script async defer src="//platform.twitter.com/widgets.js"></script>
```

Comment:
We could have a race condition here where this doesn't load before we run the code above...
Happy for now until we see an error.

What % of the browsers we support support async / defer?